script/caffe2intel-i8/convert.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2018 cTuning foundation.
+# See CK COPYRIGHT.txt for copyright details.
+#
+# SPDX-License-Identifier: BSD-3-Clause.
+# See CK LICENSE.txt for licensing details.
+#
+
+function ConvertPrototxt() {
+  PROTOTXT=$1
+  BLOB_NAME=$2
+  ITERATIONS=$3
+  INPUT_PROTOTXT=${PROTOTXT}.prototxt
+  OUTPUT_PROTOTXT=${PROTOTXT}_quantized.prototxt
+  THIS_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+  echo
+  echo "Convertion ${INPUT_PROTOTXT} ..."
+  echo
+
+  if [ -f ${INSTALL_DIR}/${INPUT_PROTOTXT} ]; then
+    rm ${INSTALL_DIR}/${INPUT_PROTOTXT}
+  fi
+  if [ -f ${INSTALL_DIR}/${OUTPUT_PROTOTXT} ]; then 
+    rm ${INSTALL_DIR}/${OUTPUT_PROTOTXT}
+  fi
+
+  python ${THIS_SCRIPT_DIR}/prepare_prototxt.py \
+    --model=${ORIGINAL_PACKAGE_DIR}/${INPUT_PROTOTXT} \
+    --target=${INSTALL_DIR}/${INPUT_PROTOTXT}
+
+  python ${CK_ENV_LIB_CAFFE}/../src/scripts/calibrator.py \
+    --root=${CK_ENV_LIB_CAFFE} \
+    --weights=${CK_ENV_MODEL_CAFFE_WEIGHTS} \
+    --model=${INSTALL_DIR}/${INPUT_PROTOTXT} \
+    --iterations=${ITERATIONS} \
+    --blob_name=${BLOB_NAME}
+
+  if [ ${PROTOTXT} == "deploy" ]; then
+    FINALIZE_SCRIPT=finalize_deploy_prototxt.py
+  else
+    FINALIZE_SCRIPT=finalize_train_val_prototxt.py
+  fi
+  python ${THIS_SCRIPT_DIR}/${FINALIZE_SCRIPT} \
+    --model=${INSTALL_DIR}/${OUTPUT_PROTOTXT}
+
+  rm ${INSTALL_DIR}/${INPUT_PROTOTXT}
+}
+
+# This path should be a module to be able importing
```

Comment:
The comment is a bit unclear. How about:
> At this path, there should be a module that can be imported.

Reply:
Changed.